tests/core/test_lane.py

```diff
@@ -4094,13 +4094,13 @@ def _w(**kwargs):
 
         # Create children works.
         result.children_with_age = _w(
-            title="Children work with set target age",
+            title="Children work with target age",
```

Comment:
SQL vs OpenSearch alphabetical sort seems to have some differences that sort the previous title differently. This change works around the difference.